### ISSUE TITLE
Error serverside 

### ISSUE THREAD

**Issue by techxpert211** [Guad/GTACoop#26]
"approve() called in wrong status; expected RespondedAwaitingApproval; got Connected
Ignoring multiple connect() most likely due to a delayed approval

This happens when my friend tries to connect to a server when im on it and vice versa.

Help.

**Comment by Guad** [comment]
Do you use any filterscripts/gamemodes? Also is that the full error message?

**Comment by techxpert211** [comment]
No, it's set to free roam. The only script that is in there is the
administration script, but that doesn't load properly. That message is the
only thing it will spew out 3 times for the last string. He can connect to
other peoples servers but it will throw the users on that server off.
